### DISCUSSION TITLE
Customisation of the plugin

### DISCUSSION THREAD

**Discussion by babulance10** [saleor/saleor#8384] [General]
Using plugins, we can create custom tables in the database and graphql API and display their fields in the UI

**Comment by PlusA2M** [comment]
I think the main method to extending the model is to use [`Metadata`](https://docs.saleor.io/docs/3.0/developer/metadata) so that your data will be related to the corresponding data like `Product`, `User`, `Sale`, etc. while exposing to the GraphQL API automatically.

**Reply by babulance10** [reply]
@PlusA2M  Thanks you.
If we want to create new table and reference to one of the product, user etc. table. Is it possible to do with plugins?
